src/machine/board_ae_rp2040.go

```diff
@@ -2,6 +2,11 @@
 
 package machine
 
+func init() {
```

Comment:
Avoid init functions in machine package always unless absolutely required. I'd like to understand the problem you are trying to solve so I can propose a better way of doing this.